### ISSUE TITLE
negative ctc_loss when training

### ISSUE THREAD

**Issue by TinBarbie** [TensorSpeech/TensorFlowASR#172]
Hi, this repo is really awesome.
I'm training deepspeech2 model with vietnamese dataset (vivos + viet_tts) and encounting an issue when the ctc_loss is negative after some batches. Is it possible to be negative? if not, any idea to fix this issue?
Thanks.
![Capture](https://user-images.githubusercontent.com/65520145/113243798-e279e880-92dd-11eb-94a7-c86cf0b816d2.JPG)

**Comment by usimarit** [comment]
@TinBarbie The loss shouldn't be negative, can you train only on vivos to see if the issue happens?

**Comment by usimarit** [comment]
Feel free to reopen if the issue still exists. Make sure your dataset contains only characters specified in vocab file.
